<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAdvancedSettingsWidgetBase</class>
 <widget class="QWidget" name="QgsAdvancedSettingsWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>610</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_26">
     <property name="title">
      <string>Advanced Settings Editor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_42">
      <item>
       <widget class="QWidget" name="mAdvancedSettingsWarning" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_44">
         <item>
          <widget class="QLabel" name="label_44">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Changes on this page are dangerous and can break your QGIS installation in various ways. Any change you make is applied immediately, without clicking the &lt;span style=&quot; font-style:italic;&quot;&gt;OK&lt;/span&gt; button.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mAdvancedSettingsEnableButton">
           <property name="text">
            <string>I will be careful, I promise!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QgsSettingsTree" name="mAdvancedSettingsEditor" native="true">
        <property name="visible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsSettingsTree</class>
   <extends>QWidget</extends>
   <header>qgssettingstree.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
